src/components/button/button.tsx

```diff
@@ -57,18 +57,24 @@ export class Button {
     @Prop({ reflect: true })
     public loading = false;
 
-    @Element()
-    private limelButton: HTMLLimelButtonElement;
+    /**
+     * Set to `true` to indicate failure instead of success when the button is
+     * no longer in the `loading` state.
+     */
+    @Prop()
+    public loadingFailed = false;
```

Comment:
Always use `reflect: true` for `@Prop`s that have a primitive value. [Why?](https://developers.google.com/web/fundamentals/web-components/best-practices#aim-to-keep-primitive-data-attributes-and-properties-in-sync,-reflecting-from-property-to-attribute,-and-vice-versa.)

```suggestion
    @Prop({ reflect: true })
    public loadingFailed = false;
```